.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Thank you for sending in this pull request. Please make sure you take a look at the [contributing file](CONTRIBUTING.md). Here's a few things for you to consider out on this pull request:
+
+- [ ] Include a good description of the workflow.
+- [ ] Links to the language or tool will be nice (unless its really obvious).
+- [ ] For the `*.yml` file, please include a matching `ci/properties/*.properties.json` file.
+- [ ] Does not use an Action that isn't in the `actions` organization.
+- [ ] Does not send data to any 3rd party service except for the purposes of installing dependencies.
+- [ ] Does use a paid service or product.
```

Comment:
- [ ] Use title case for the names of workflows and steps, for example "Run tests".

Reply:
```yaml
- [ ] The name of CI workflows should only be the name of the language or platform: for example "Go" (not "Go CI" or "Go Buil
- [ ] CI workflows should run `on: [push]`.  Packaging workflows should run:

    on:
        release:
            types: [created]
```

Reply:
Oh, also, should this be "Does not use a paid service or product"?